files/default/sql/ddl/updates/1.4.0.sql

```diff
@@ -44,4 +44,28 @@ DROP TABLE IF EXISTS `hopsworks`.`tf_lib_mapping`;
 ALTER TABLE `hopsworks`.`project` ADD COLUMN `docker_image` varchar(255) COLLATE latin1_general_cs DEFAULT NULL;
 ALTER TABLE `hopsworks`.`project` DROP COLUMN `conda_env`;
 
-ALTER TABLE `hopsworks`.`conda_commands` DROP COLUMN `docker_image`;
\ No newline at end of file
+ALTER TABLE `hopsworks`.`conda_commands` DROP COLUMN `docker_image`;
+
+ALTER TABLE `conda_commands` DROP FOREIGN KEY `user_fk`, ADD CONSTRAINT `user_fk_conda` FOREIGN KEY (`user_id`) REFERENCES `users` (`uid`) ON DELETE CASCADE ON UPDATE NO ACTION;
+
+ALTER TABLE `feature_group` DROP FOREIGN KEY `on_demand_feature_group_fk`, ADD CONSTRAINT `on_demand_feature_group_fk2` FOREIGN KEY (`on_demand_feature_group_id`) REFERENCES `on_demand_feature_group` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;
+
+ALTER TABLE `schemas`  DROP FOREIGN KEY `project_idx`, ADD  CONSTRAINT `project_idx_schemas` FOREIGN KEY (`project_id`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;
+
+ALTER TABLE `subjects`  DROP FOREIGN KEY `project_idx`, ADD CONSTRAINT `project_idx_subjects` FOREIGN KEY (`project_id`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;
+
+ALTER TABLE `subjects_compatibility`  DROP FOREIGN KEY `project_idx`,  ADD CONSTRAINT `project_idx_sub_comp` FOREIGN KEY (`project_id`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;
+
+ALTER TABLE `project_topics`  DROP FOREIGN KEY `project_idx`,  ADD CONSTRAINT `project_idx_proj_topics` FOREIGN KEY (`project_id`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;
+
+ALTER TABLE `rstudio_settings`  DROP FOREIGN KEY `RS_FK_USERS`, ADD CONSTRAINT `FK_262_310` FOREIGN KEY (`team_member`) REFERENCES `users` (`email`) ON DELETE CASCADE ON UPDATE NO ACTION;
+
+ALTER TABLE `rstudio_settings`  DROP FOREIGN KEY `RS_FK_PROJS`, ADD CONSTRAINT `FK_284_312` FOREIGN KEY (`project_id`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION;
```

Comment:
Here you inverted the drop and the add names  - should be dropped `FK_262_310` ...

Reply:
Yes, fixd it